src/main/scala/cqrs/domain/orders/Order.scala

```diff
@@ -1,13 +1,15 @@
 package cqrs.domain.orders
 
-import akka.actor.{ Status, ActorLogging, Props }
+import akka.actor.{ReceiveTimeout, Status, ActorLogging, Props}
 import akka.persistence.PersistentActor
+import cqrs.domain.orders.OrderCommandHandler.UnknownOrderException
```

Comment:
You can change `import cqrs.domain.orders.OrderCommandHandler.UnknownOrderException`
to `import OrderCommandHandler.UnknownOrderException`